src/oc/web/actions/user.cljs

```diff
@@ -378,6 +378,16 @@
 (defn user-profile-reset []
   (dis/dispatch! [:user-profile-reset]))
 
+(defn user-role []
```

Comment:
This is user data related and belongs in the user store instead of action.

Reply:
@thepug that's right but i can't access the user store directly from the UI or i'll break the pattern (check the graph, from the component i can only call an action creator, ie a user action).
WYT?